### ISSUE TITLE
Recent folders not showing up

### ISSUE THREAD

**Issue by JimmyPeter** [alainfrisch/nostalgy#53]
When pressing S or G, I won't get the recent folders list. Instead I only see the last folder I used with S or G, but I can't select it by pressing up / down. Instead, I have to start typing again, even if I want to move / go to the same folder.

Thunderbird: 60.0
Nostalgy: 0.2.36

**Comment by dsteinkopf** [comment]
I also have this problem. As a kind of workaround I press left followed by the up key.

**Comment by JimmyPeter** [comment]
Oh nice, that will speed things up until this is fixed. Thanks for the info!

**Comment by JoelAzaria** [comment]
@dsteinkopf Not sure if your descript matches what I see.  

If I save a message, I type a few characters and get a "focused list".  Becuase I use distinct project numbers that list is usually two lines (the folder and the tag).  I use the arrows to select the folder for example, my email saves there.
Then I go to repeat.  I hit save and the arrow key is blocked.   But I'm *looking* at the same list of two destinations as before.   I hit left then up.  I'm no longer seeing the two destinations.  I have a list of 10 or so items.  Is that what you're seeing too?

If so then you can achieve the same thing btw by just start typing.  When the arrows are locked just type your destination.  Since for me I wanted the same dest as before I can see it and I just re-type.  Left/up actually makes me lose that visual queue and (for me anyway) I'm better of with it.

Just wanna verify that we're seeing the same thing.   This TB60 mess seems to get weirder the further down the rabbit hole we go..

Because actually I came here to report somewhere (maybe relevant here so I'll put it here), that after I've saved a message to a folder, the status bar that shows where 'Shift+S" would go (which *should* be the last saved dest) doesn't show the last destination.   Like it's not updating where it last saved a message.

Could you maybe verify that for me please?
Thx.

**Comment by dsteinkopf** [comment]
Yes, it's the same here: After pressing left and up I see the list of (about) 10 recently used destinations. 
And yes, retyping gives me this "search result" again.
But in my case I prefer not to type the search string again, so I prefer left-up-up-[n times]-enter.

**Comment by JoelAzaria** [comment]
@dsteinkopf thing is, in my case, that list of about 10, doesn't seem to include my recently used dest.  Or at least not always.  I commented on something similar at the bottom of #46, which is basically that the shift+s shortcut isn't always updating to the last dest and now I'm wondering if that's tied to me not seeing the last dest in that list of ten.  Gonna have to really pay close close attention and see if I can figure wth is actually going on.  Definitely something 'off' and definitely intermittent.

Since it is intermittent, please see if you can notice any similar behavior at any time on your end.  thx.

**Comment by h271fa** [comment]
#62
